pkg/api/types.go

```diff
@@ -401,6 +401,10 @@ const (
 	ClaimPending PersistentVolumeClaimPhase = "Pending"
 	// used for PersistentVolumeClaims that are bound
 	ClaimBound PersistentVolumeClaimPhase = "Bound"
+	// used for PersistentVolumeClaims that lost their underlying
+	// PersistentVolume. The claim was bound to a PersistentVolume and this
+	// volume does not exist any longer and all data on it was lost.
+	ClaimLost PersistentVolumeClaimPhase = "Lost"
```

Comment:
I think it's fundamentally different than `Released`. Released PV successfully fulfilled its purpose without errors, while the Lost claim is in very bad error state.

I think someone suggested `Failed` state, but there were some good arguments I don't remember why it is bad.